src/commands.js

```diff
@@ -78,6 +78,60 @@ DeregisterHandler = async (ctx) => {
     await ctx.otherBot.telegram.sendMessage(telegramId, messages.DeregisterSuccess)
 }
 
+ConfirmHandler = async (ctx) => {
+    if (!ctx.isRegistered) {
+        return ctx.reply(messages.NotRegistered)
+    }
+    var person = ctx.person
+    if (person.confirm) {
```

Comment:
If i'm understanding your intention correctly (boolean var for today's theme), I would instead name this `ctx.person.confirmedToday`
- change variable name to be more informative
- inlined ctx